dao-go/assignments_test.go

```diff
@@ -2,14 +2,337 @@ package dao_test
+	proposer := env.Members[0]
+	assignee := env.Members[1]
+	closer := env.Members[2]
+
+	role1Doc := CreateRole(t, env, proposer, closer, role1)
+	t.Run("Test Adjust assignment commitment", func(t *testing.T) {
+
+		tests := []struct {
+			name       string
+			roleTitle  string
+			title      string
+			role       docgraph.Document
+			assignment string
+			husd       string
+			hypha      string
+			hvoice     string
+			usd        string
+		}{
+			{
+				name:       "role1 - 100% 100%",
+				roleTitle:  "Underwater Basketweaver",
+				title:      "Underwater Basketweaver - Atlantic",
+				role:       role1Doc,
+				assignment: assignment1,
+				husd:       "0.00 HUSD",
+				hypha:      "759.75 HYPHA",
+				hvoice:     "6078.02 HVOICE",
+				usd:        "3039.01 USD",
+			},
+		}
+
+		for _, test := range tests {
+
+			t.Log("\n\nStarting test: ", test.name)
+
+			_, err := dao.ProposeAssignment(env.ctx, &env.api, env.DAO, proposer.Member, assignee.Member, test.role.Hash, env.Periods[0].Hash, test.assignment)
+			assert.NilError(t, err)
+
+			// retrieve the document we just created
+			proposal, err := docgraph.GetLastDocumentOfEdge(env.ctx, &env.api, env.DAO, eos.Name("proposal"))
+			assert.NilError(t, err)
+		
+			ballot, err := proposal.GetContent("ballot_id")
+			assert.NilError(t, err)
+			voteToPassTD(t, env, ballot.Impl.(eos.Name))
+
+			//Wait 1 Period to close the proposal and test the special 
+			//case when approved time overlaps in the first period
+			t.Log("Waiting for a period to lapse...")
+			pause(t, env.PeriodPause, "", "Waiting...")
+
+			_, err = dao.CloseProposal(env.ctx, &env.api, env.DAO, closer.Member, proposal.Hash)
+			assert.NilError(t, err)
+
+			assignment, err := docgraph.GetLastDocumentOfEdge(env.ctx, &env.api, env.DAO, eos.Name("assignment"))
+
+			var firstPeriodStartSecs int64
+			var firstPeriodEndSecs int64
+			//Get starting period to calculate half period duration
+			{
+				periodHash, err := assignment.GetContent("start_period")
+				assert.NilError(t, err)
+				period, err := docgraph.LoadDocument(env.ctx, &env.api, env.DAO, periodHash.String());
+				assert.NilError(t, err)
+				startTime, err := period.GetContent("start_time")
+				assert.NilError(t, err)
+				firstPeriodStartSecs = int64(startTime.Impl.(eos.TimePoint))/1000000
+				nextPeriods, err := docgraph.GetEdgesFromDocumentWithEdge(env.ctx, &env.api, env.DAO, period, eos.Name("next"))
+				assert.NilError(t, err)
+				nextPeriod, err := docgraph.LoadDocument(env.ctx, &env.api, env.DAO, nextPeriods[0].ToNode.String())
+				assert.NilError(t, err)
+				startTime, err = nextPeriod.GetContent("start_time")
+				assert.NilError(t, err)
+				firstPeriodEndSecs = int64(startTime.Impl.(eos.TimePoint))/1000000
+			}
+
+			//Create Adjustment 2.5 Periods after start period
+			CreateAdjustmentAfter(int64(50), 
+														firstPeriodStartSecs, 
+														env.PeriodDuration * 5 / 2,
+														&assignment,
+														&assignee, env, t)
```

Comment:
Could you pull this block (and the ones bellow) to the left? It's more complicated to read them like this.